### PR TITLE
chore/cli: add paseo config

### PR DESCRIPTION
**Pull request by kanishkatn**
## Changes
Adds the paseo chain-spec to the project and supports it in the cli.

## Issues

#3671 

## Primary Reviewer

@EclesioMeloJunior

### CODE REVIEW

File: chain/paseo/defaults.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	cfg "github.com/ChainSafe/gossamer/config"
+)
+
+var (
+	// defaultName Default node name
+	defaultName = "Paseo"
+	// defaultID Default chain ID
+	defaultID = "paseo"
+	// defaultBasePath Default node base directory path
+	defaultBasePath = "~/.gossamer/paseo"
```
Comment (q9f) [inline chain/paseo/defaults.go:16]: This should be `$XDG_DATA_HOME`, I'll create a new issue

https://github.com/ChainSafe/gossamer/issues/3678

**Review by q9f — APPROVED** [review]

File: chain/paseo/defaults.go (hunk 1 above)
Comment (kanishkatn) [reply chain/paseo/defaults.go:16]: Gonna have to make a small change for this #3211

Comment (kanishkatn) [reply chain/paseo/defaults.go:16]: Thank you!

**Review by EclesioMeloJunior — APPROVED** [review]

**Review by dimartiro — APPROVED** [review]

**Review by q9f — CHANGES_REQUESTED** [review]
please remove the XDG changes before merging this

**Comment by kanishkatn** [comment]
> please remove the XDG changes before merging this

It's done!

**Comment by kanishkatn** [comment]
Moving out the renaming to a different PR to sort out the CI issues. My bad. 

@EclesioMeloJunior @dimartiro Paging to take another look at this!

**Review by q9f — APPROVED** [review]

**Review by EclesioMeloJunior — APPROVED** [review]

**Pull request merged**